dotenv/godotenv.go

```diff
@@ -131,75 +124,13 @@ func Read(filenames ...string) (map[string]string, error) {
 	return ReadWithLookup(nil, filenames...)
 }
 
-// Unmarshal reads an env file from a string, returning a map of keys and values.
-func Unmarshal(str string) (map[string]string, error) {
-	return UnmarshalBytes([]byte(str))
-}
-
-// UnmarshalBytes parses env file from byte slice of chars, returning a map of keys and values.
-func UnmarshalBytes(src []byte) (map[string]string, error) {
-	return UnmarshalBytesWithLookup(src, nil)
-}
-
```

Comment:
Should we leave these exported methods in case someone else is using them?

Reply:
Agreed with Nick, we should just update their internal implementation if needed
If we want to remove them in a near future, we should mark them as `deprecated` first

Reply:
Ahhh yeah, went overboard here, will restore these!

(FYI The only reason I started deleting code is that I found out there were actually 2x very similar but different parsing code paths! And the tests were focused on one while the actual impl code was using the other 🙈 )

Reply:
Re-introduced in c23e7ac3dbcfd58fab23a9666792face547d44c2!

I didn't deprecate these as they're not really problematic, I just went overboard by accident 😅 

(Not the same story for `Marshal` / `Write` tho!)